code/modules/mob/living/carbon/examine.dm

```diff
@@ -196,19 +196,16 @@
 			msg += "<span class='warning'>[p_they(TRUE)] appear[p_s()] to have committed suicide... there is no hope of recovery.</span>\n"
 		if(!just_sleeping)
 			msg += "<span class='deadsay'>[p_they(TRUE)] [p_are()] limp and unresponsive; there are no signs of life"
-			if(get_int_organ(/obj/item/organ/internal/brain))
-				if(!key)
-					var/foundghost = FALSE
-					if(mind)
-						for(var/mob/dead/observer/G in GLOB.player_list)
-							if(G.mind == mind)
-								foundghost = TRUE
-								if(G.can_reenter_corpse == 0)
-									foundghost = FALSE
-								break
-					if(!foundghost)
-						msg += " and [p_their()] soul has departed"
-			msg += "...</span>\n"
+			if(get_int_organ(/obj/item/organ/internal/brain) && !key)
+				var/foundghost = FALSE
+				if(mind)
+					for(var/mob/dead/observer/G in GLOB.player_list)
+						if(G.mind == mind && G.can_reenter_corpse)
+							foundghost = TRUE
+						break
+				if(!foundghost)
+					msg += " and [p_their()] soul has departed"
+				msg += "...</span>\n"
```

Comment:
This needs to show up even if there is no brain
```suggestion
			msg += "...</span>\n"
```

Reply:
I am unable to count apparently.